stonesoup/reader/video.py

```diff
@@ -0,0 +1,227 @@
+# -*- coding: utf-8 -*-
+"""Video readers for Stone Soup.
+
+This is a collection of video readers for Stone Soup, allowing quick reading
+of video data/streams.
+"""
+
+from abc import abstractmethod
+import datetime
+import numpy as np
+try:
+    import ffmpeg
+    import moviepy.editor as mpy
+except ImportError:
+    print("Usage of video processing classes requires that the optional"
+          "package dependencies 'moviepy' and 'ffmpeg-python' are installed. "
+          "This can be achieved by running 'python -m pip install .[video]'")
```

Comment:
I'm thinking this  should be `python -m pip install stonesoup[video]`, as it'll allow those who aren't installing from local clone to install.

Also, I'm wondering if this message should be part of an "ImportError" rather than a separate `print` statement. Its a bit clearer that there's an error, and also goes to _stderr_ rather than _stdout_. Something like:
```python
except ImportError as error:
    raise ImportError(
          "Usage of video processing classes requires that the optional"
          "package dependencies 'moviepy' and 'ffmpeg-python' are installed. "
          "This can be achieved by running 'python -m pip install .[video]'") \
          from error

```